### PR TITLE
Add KeepAliveTimeout to GrpcChannelOptions

### LINKED ISSUE #433: Expose keepalive timeout (as well as keepalive time) in GrpcChannelOptions
gRPC option name: "grpc.keepalive_timeout_ms"

### PR DESCRIPTION
**Pull request by jskeet**
Fixes #433

### CODE REVIEW

File: Google.Api.Gax.Grpc/GrpcChannelOptions.cs [hunk 1]
```diff
@@ -201,6 +221,7 @@ IReadOnlyList<CustomOption> MergeCustomOptions(IReadOnlyList<CustomOption> origi
             other is object &&
             EnableServiceConfigResolution == other.EnableServiceConfigResolution &&
             KeepAliveTime == other.KeepAliveTime &&
+            KeepAliveTimeout == other.KeepAliveTimeout &&
```
Comment (amanda-tarafa) [inline Google.Api.Gax.Grpc/GrpcChannelOptions.cs:224]: Should probably be added to `GetHashCode` as well.
And there's an Equality test where this should be added as well.

**Review by amanda-tarafa — APPROVED** [review]
LGTM, but one change needed.

File: Google.Api.Gax.Grpc/GrpcChannelOptions.cs (hunk 1 above)
Comment (jskeet) [reply Google.Api.Gax.Grpc/GrpcChannelOptions.cs:224]: Ah, well done - thanks for spotting that. Will fix now in a separate commit for you to check, then will merge those two together.

**Review by amanda-tarafa — APPROVED** [review]
Yep, LGTM!

**Pull request merged**